benches/serialization.rs

```diff
@@ -1,25 +1,25 @@
-use criterion::{criterion_group, criterion_main, Criterion};
-use nickel_lang::program::Program;
+use criterion::{criterion_main, Criterion};
+use nickel_lang_utilities::ncl_bench_group;
 use pprof::criterion::{Output, PProfProfiler};
-use std::io::Cursor;
+
 use std::path::PathBuf;
 
-fn round_trip(c: &mut Criterion) {
+fn serialization_path(subpath: &str) -> String {
```

Comment:
I think this function is not used anymore (indeed, there is a compilation warning).